template.tpl

```diff
@@ -96,12 +96,156 @@ ___TEMPLATE_PARAMETERS___
     "name": "eventType",
     "type": "SELECT"
   },
+  {
+    "type": "GROUP",
+    "name": "eventMetadataGroup",
+    "displayName": "Additional Event Information",
+    "groupStyle": "NO_ZIPPY",
+    "subParams": [
+      {
+        "type": "SIMPLE_TABLE",
+        "name": "eventMetadataParams",
+        "displayName": "",
+        "simpleTableColumns": [
+          {
+            "defaultValue": "",
+            "displayName": "Parameter Name",
+            "name": "name",
+            "type": "SELECT",
+            "selectItems": [
+              {
+                "value": "itemCount",
+                "displayValue": "Item Count"
+              },
+              {
+                "value": "value",
+                "displayValue": "Event Value"
+              },
+              {
+                "value": "currency",
+                "displayValue": "Currency"
```

Comment:
`currency` is expected to be a string, but must be exactly 3 characters long.  Otherwise the pixel JavaScript will filter it out.  Can we perform that check below?  Say, pad 1- or 2-character strings with spaces, and trim any string with 4+ characters?

Reply:
I'm not really sure on this since I think GTM should simply pass on whatever the advertiser has configured to our pixel (just like if they had manually set up). I'm not sure if we should add much business logic here. Open to hearing others' thought though

Reply:
Sounds good!  Keep it simple.